Eatery Blue/UI/EateryCard/EateryMediumCardContentView.swift

```diff
@@ -15,8 +17,12 @@ class EateryMediumCardContentView: UIView {
 
     let titleLabel = UILabel()
     let subtitleLabel = UILabel()
-    let favoriteImageView = UIImageView()
-
+    let favoriteButton = ButtonView(content: UIImageView())
+    
+    let alertView = EateryCardAlertView()
+    
+    private var cancellables: Set<AnyCancellable> = []
```

Comment:
```suggestion
    private var cancellables = Set<AnyCancellable>()
```
Not sure if that works but we want to keep it consistent with what we have above.